pages/FilmFinder.js

```diff
@@ -28,23 +30,50 @@ const search = searchInput => {
     }
   })
 }
+
+const onClose = () => {
+  setSelectedFilm(null);
+}
+
+const renderSelectedFilm = () => {
+  if (selectedFilm) {
+        return (
+          <div className={styles.filmDetail} >
+            <FilmDetail className={styles.filmDetail} film={selectedFilm}/>
+            <button className={styles.closeButton}
+                width="50"
+                alt={"close button"}
+                onClick={onClose}>
+              close
+            </button>
+          </div>
```

Comment:
I would place all of this inside the `FilmDetail` component, that would clean this file up nicely and then you are keeping the styling for `FilmDetail` all in once place rather than across multiple CSS files. 

Then this becomes:

`return <FilmDetail film={selectedFilm} onClose={onClose} />`.

Also the `FilmDetail` component doesn't have a prop `className` so that won't do anything! I would suggest adding `prop-types` to this project to ensure you're are not creating any bugs by passing wrong data types!